ddtrace/tracer/tracer.go

```diff
@@ -138,6 +144,8 @@ func Start(opts ...StartOption) {
 	cfg.HTTP = t.config.httpClient
 	cfg.ServiceName = t.config.serviceName
 	appsec.Start(appsec.WithRCConfig(cfg))
+	go hostname.Get() // Prime the hostname cache
```

Comment:
Why put this in a go routine?

Reply:
`hostname.Get()` already starts a go routine to run the fetching logic. Perhaps this go routine here is redundant / unnecessary ?